ceph-releases/jewel/ubuntu/14.04/daemon/entrypoint.sh

```diff
@@ -79,25 +79,19 @@ function dev_part {
 }
 
 function osd_trying_to_determine_scenario {
-  if [[ ! -d /var/lib/ceph/osd || -n "$(find /var/lib/ceph/osd -prune -empty)" ]]; then
```

Comment:
If you remove this, the next if is wrong.
If we pass this `if` this means `/var/lib/ceph/osd` has some content, thus the next condition can safely assume if no  `OSD_DEVICE` is declared that we can try `osd_directory`.